src/wp-includes/default-filters.php

```diff
@@ -533,6 +533,9 @@
 add_action( 'delete_attachment', '_delete_attachment_theme_mod' );
 add_action( 'transition_post_status', '_wp_keep_alive_customize_changeset_dependent_auto_drafts', 20, 3 );
 
+// Block Theme Previews
```

Comment:
```suggestion
// Block Theme Previews. 
```

Add missing end period.